devices/tuya/ZY-M100_human_breathing_presence.json

```diff
@@ -0,0 +1,286 @@
+{
+  "schema": "devcap1.schema.json",
+  "manufacturername": [
+    "_TZE200_ztc6ggyl",
+    "_TZE204_ztc6ggyl"
+  ],
+  "modelid": [
+    "TS0601",
+    "TS0601"
+  ],
+  "product": "Human Presence Detector - ZY-M100",
+  "sleeper": true,
+  "status": "Silver",
+  "subdevices": [
+    {
+      "type": "$TYPE_LIGHT_LEVEL_SENSOR",
+      "restapi": "/sensors",
+      "uuid": [
+        "$address.ext",
+        "0x01",
+        "0x0400"
+      ],
+      "items": [
+        {
+          "name": "attr/lastseen"
+        },
+        {
+          "name": "attr/manufacturername"
+        },
+        {
+          "name": "attr/modelid"
+        },
+        {
+          "name": "attr/name"
+        },
+        {
+          "name": "attr/swversion",
+          "refresh.interval": 84000,
+          "read": {
+            "at": "0x0001",
+            "cl": "0x0000",
+            "ep": 1,
+            "fn": "zcl"
+          },
+          "parse": {
+            "at": "0x0001",
+            "cl": "0x0000",
+            "ep": 1,
+            "fn": "zcl",
+            "script": "tuya_swversion.js"
+          }
+        },
+        {
+          "name": "attr/type"
+        },
+        {
+          "name": "attr/uniqueid"
+        },
+        {
+          "name": "config/on"
+        },
+        {
+          "name": "config/reachable"
+        },
+        {
+          "name": "state/lastupdated"
+        },
+        {
+          "name": "state/lightlevel",
+          "read": {
+            "fn": "none"
+          },
+          "parse": {
+            "dpid": 104,
+            "eval": "const lux = Attr.val; if (lux > 0 && lux < 0xffff) {Item.val = Math.round(10000 * Math.log10(lux) + 1);};",
+            "fn": "tuya"
+          },
+          "default": 0
+        },
+        {
+          "name": "state/lux",
+          "description": "The current light intensity in Lux (max is 2000)",
+          "read": {
+            "fn": "none"
+          },
+          "parse": {
+            "dpid": 104,
+            "eval": "Item.val = Attr.val;",
+            "fn": "tuya"
+          },
+          "default": 0
+        }
```

Comment:
Hi. Not sure to understand whole comment.
This device only present luminance measure in lux. Others are of no meaning for it. I added lightlevel with this formula following a request in the initial issue but with no real use for it. It can be removed because not needed. Dark and daylight are of no meaning in this device.

Reply:
Think i've understood and modified.